todo/views.py

```diff
@@ -0,0 +1,61 @@
+from re import M
+from django.shortcuts import render
+from rest_framework import mixins
+from rest_framework import viewsets
+from rest_framework import permissions
+from rest_framework import status
+from rest_framework.response import Response
+
+
+from .models import Todo
+from .serilizers import TodoSerializer
+# Create your views here.
+
+
+class ShowAllTodos(mixins.ListModelMixin, viewsets.GenericViewSet):
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def list(self, request):
+        queryset = Todo.objects.filter(user=request.user)
+        data = [TodoSerializer(todo).data for todo in queryset]
+        return Response(data, status=status.HTTP_200_OK)
+
+
+class CreateTodo(mixins.UpdateModelMixin, viewsets.GenericViewSet):
```

Comment:
Use TodoSerializer for validation:

```
serializer = TodoSerializer(data=request.data)
serializer.is_valid(raise_exception=True)
```

This will give you:
`serializer.validated_data`

which you can use to create your entity instance:
Todo.objects.create(**serializer.validated_data)